### PR TITLE
Move lock file from /tmp to /var/run/lock. Do not run logrotate immediately on boot.

### LINKED ISSUE #3249: race condition with systemd-tmpfiles-clean.timer
On VCH boot systemd runs systemd-tmpfiles-clean.timer that cleans up /tmp directory. It results in race condition between logrotate runner that stores its file lock and config in /tmp directory.

### PR DESCRIPTION
**Pull request by vburenin**
Fixed #3249

### CODE REVIEW

File: pkg/filelock/flock.go [hunk 1]
```diff
@@ -36,7 +36,7 @@ type FileLock struct {
 func NewFileLock(lockName string) *FileLock {
 	return &FileLock{
 		LockName: lockName,
-		LockFile: filepath.Join(os.TempDir(), lockName),
+		LockFile: filepath.Join("/var/run/lock", lockName),
```
Comment (hmahmood) [inline pkg/filelock/flock.go]: Does /var/run/lock exist always?

Comment (vburenin) [inline pkg/filelock/flock.go]: yes, it is created during boot.

**Comment by caglar10ur** [comment]
LGTM

**Comment by hickeng** [comment]
LGTM

**Pull request merged**